README.md

```diff
@@ -44,7 +44,7 @@ Before using the salesforce_ar_sync gem you must ensure you have the databasedot
 properly. Make sure each of the models you wish to sync are materialized.
 
```

Comment:
I also recommend updating the README file in general, there should be no need for the databasedotcom gem anymore. We should update the documentation to reflect that as well. I see some references to it.